spec/unit/changed_managed_files_inspector_spec.rb

```diff
@@ -28,7 +28,7 @@
   subject {
     inspector = ChangedManagedFilesInspector.new(system, description)
 
-    allow(system).to receive(:run_script).with("changed_managed_files.sh", anything()).and_return(rpm_result)
+    allow(system).to receive(:run_script).with("inspect_files.sh", any_args()).and_return(rpm_result)
```

Comment:
Line is too long. [101/100]<br>Do not use parentheses for method calls with no arguments.